deepspeed-mii/config.yaml

```diff
@@ -0,0 +1,48 @@
+spec_version: '2.0'
```

Comment:
We've recently decided to keep the config.yaml files short. If the value is same as default value we don't set it. I feel most of the settings here can be removed. For reference default values are here: https://github.com/basetenlabs/truss/blob/main/truss/truss_config.py

Reply:
Cleaned it up, take a look if anything extra left.